zebrad/tests/common/lightwalletd.rs

```diff
@@ -103,33 +105,62 @@ where
 
         // By default, launch a working test instance with logging,
         // and avoid port conflicts.
-        let mut args: Vec<_> = vec![
-            // the fake zcashd conf we just wrote
-            "--zcash-conf-path",
-            default_config_path
-                .as_path()
-                .to_str()
-                .expect("Path is valid Unicode"),
-            // the lightwalletd cache directory
-            //
-            // TODO: create a sub-directory for lightwalletd
+        let mut args_map = IndexMap::new();
```

Comment:
Question: What if some arguments are meant to be repeated?

I had a look at the lightwalletd implementation, and it seems like repeated arguments are not supported:
https://github.com/zcash/lightwalletd/blob/631bb16404e3d8b045e74a7c5489db626790b2f6/cmd/root.go#L313

Can you update the trait method docs to say that?

Reply:
I added a note about that in the documentation of the new `Arguments` type.